src/main/java/frc/robot/commands/ArmMoveToShooterCommand.java

```diff
@@ -24,8 +24,9 @@ public void initialize() {
     @Override
     public void execute() {
         double output = pidController.calculate(armSystem.getAbsEncoderPositionDegrees(),  RobotMap.ARM_SHOOTER_ANGLE);
+        pidController.setIZone(RobotMap.ARM_PID_I_ZONE);
```

Comment:
don't change Izone during calculation,  set it in the ctor once.

Reply:
done